dockerfile-suse.template

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM __BASE_IMAGE__
+ARG ARCH
+ARG DISTRO_NAME
+ARG DISTRO_NICK
+ARG DISTRO_RELEASE
+ARG AT_VERSION
+ARG EXTRA
+ARG MINOR
+ARG REPO
+
+WORKDIR /root
+
+RUN zypper refresh \
+    && zypper install -y __BASE_PACKAGES__ \
+    && zypper clean
+
+RUN rpm --import ${REPO}/${DISTRO_NAME}/SLES_${DISTRO_NICK}/gpg-pubkey-6976a827-5164221b \
+    && zypper addrepo ${REPO}/suse/SLES_15/ AT \
+    && zypper install -y __AT_PACKAGES__
```

Comment:
Why do you perform `zypper clean` before the later `zypper install`, and not after?

Reply:
@ThinkOpenly Good point. I'll move it to after the second install command.